backend/.gitignore

```diff
@@ -1,18 +1,26 @@
 # Byte-compiled / optimized / DLL files
 __pycache__/
 *.py[cod]
-*$py.class
 
-# C extensions
```

Comment:
I fail to see why any changes need to be made to the .gitignore based on the changes in your PR.